src/actions/historyActions.js

```diff
@@ -424,7 +424,13 @@ export const setHistoryTransactionStatusByHashAction = (transactionHash: string,
   };
 };
 
-export const viewTransactionOnBlockchainAction = (chain: Chain, event: Event) => {
+export type TransactionToView = {
+  hash?: string,
+  batchHash?: string,
+  fromAddress?: any,
+}
+
+export const viewTransactionOnBlockchainAction = (chain: Chain, event: Event | TransactionToView) => {
```

Comment:
```suggestion
export const viewTransactionOnBlockchainAction = (chain: Chain, transaction: TransactionToView) => {
```

Could we avoid union type here, using `TransactionToView` should be enough as it repeats the required fields from `Event`.

nit: Also pls consider renaming to `TransactionInfo`, as TransactionToView seems to suggest it's a complete transaction object.